modules/3d-tiles/src/types.ts

```diff
@@ -15,7 +15,7 @@ export type B3DMContent = {
   cartographicOrigin: Vector3;
   featureIds?: number[] | null;
   featureTableBinary?: Uint8Array;
-  featureTableJson?: BatchTableJson;
+  featureTableJson?: PropertyTableJson;
```

Comment:
The field is called `featureTable` and the type is called `PropertyTable`?

Reply:
Renamed it to `FeatureTableJson`